crudApp/src/reducers/Crud.js

```diff
@@ -0,0 +1,37 @@
+//A single reducer changeTheState which defines how to perform actions based on the action type.
+
+const initialState = [{
+  id:"",
+  Name:"",
+  Age:"",
+}];
+
+const changeTheState = (state = initialState, action) => {
+  switch (action.type) {
+    case "ADD":
+      return [
+        ...state,
+        {
+          id: action.payloadId,
+          Name: action.payloadName,
+          Age: action.payloadAge
+        }
+      ];
+    case "EDIT": {
```

Comment:
create a constant file and use those constants here and in your actions
e.g
export const USER = {
ADD : 'ADD',
DELETE: 'DELETE'
};